content/articles/Factor-that-affects-the-performance-deep-learning-systems/index.md

```diff
@@ -0,0 +1,114 @@
+### INTRODUCTION
+Deep learning is a type of artificial intelligence that simulates how humans learn specific sorts of information. It's a branch of AI and machine learning. This technology has been used to create many self-driving cars and smart digital devices.
```

Comment:
This looks like too generalized introduction about DL. I would suggest to rephrase it with a better introduction, more related to the deep learning performance.

Talk about performance in general, mention how deep learning models perform currently, how it can be improved, how should it be done, state any relevant examples.

You can cover each of the above points in 1 - 2 lines